CHANGES.rst

```diff
@@ -12,6 +12,10 @@ esa.jwst
 
 Service fixes and enhancements
 ------------------------------
+esa.xmm_newton
+^^^^^^^^^^^^^^
+
+- Add option to download proprietary data [#2251]
```

Comment:
0.4.5 has already been released so the change log entry needs to be updated.

Reply:
Okay, so what changes are needed exactly? Do I need to change 0.4.5 to 0.4.6 and then only leave in the xmm_newton part and remove the rest?